src/test/java/com/example/enterprise/MoneyManagerApplicationTests.java

```diff
@@ -33,9 +29,9 @@ void contextLoads() {
 
     }
 
-//    Add income
+    //    Add income
```

Comment:
Fixed the indentation for clean code practices. 

Indentation practices can be found in this link: https://www.baeldung.com/java-clean-code